app/code/Magento/Sales/etc/webapi.xml

```diff
@@ -10,271 +10,272 @@
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::shipment" />
         </resources>
     </route>
     <route url="/V1/shipment/:id/emails" method="POST">
         <service class="Magento\Sales\Api\ShipmentManagementInterface" method="notify"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::shipment" />
         </resources>
     </route>
     <route url="/V1/shipment/track" method="POST">
         <service class="Magento\Sales\Api\ShipmentTrackRepositoryInterface" method="save"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::shipment" />
         </resources>
     </route>
     <route url="/V1/shipment/track/:id" method="DELETE">
         <service class="Magento\Sales\Api\ShipmentTrackRepositoryInterface" method="deleteById"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::shipment" />
         </resources>
     </route>
     <route url="/V1/shipment/" method="POST">
         <service class="Magento\Sales\Api\ShipmentRepositoryInterface" method="save"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::shipment" />
         </resources>
     </route>
     <route url="/V1/shipment/:id/label" method="GET">
         <service class="Magento\Sales\Api\ShipmentManagementInterface" method="getLabel"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::shipment" />
         </resources>
     </route>
     <route url="/V1/order/:orderId/ship" method="POST">
         <service class="Magento\Sales\Api\ShipOrderInterface" method="execute"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::ship" />
         </resources>
     </route>
     <route url="/V1/orders/" method="POST">
         <service class="Magento\Sales\Api\OrderRepositoryInterface" method="save"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::create" />
         </resources>
     </route>
     <route url="/V1/transactions/:id" method="GET">
         <service class="Magento\Sales\Api\TransactionRepositoryInterface" method="get"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::transactions" />
         </resources>
     </route>
     <route url="/V1/transactions" method="GET">
         <service class="Magento\Sales\Api\TransactionRepositoryInterface" method="getList"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::transactions" />
```

Comment:
I think `Magento_Sales::transactions_fetch` is more appropriate for viewing transactions (read only).